docker-compose.yml

```diff
@@ -81,5 +81,22 @@ services:
     ports:
       - "5900"
     shm_size: 2g
+    links:
+      - "addons-frontend:olympia.local.test"
+      - "nginx:olympia.test"
+
+  addons-frontend:
+    <<: *env
+    environment:
+      - NODE_APP_INSTANCE=amo
+      - NODE_ENV=development
+      - API_HOST=http://olympia.test
+      - HOSTNAME=functional.test
+    ports:
+      - "4000"
+    expose:
+      - "4000"
+    image: addons/addons-frontend
+    command: yarn build ; yarn start
```

Comment:
Shouldn't this be `yarn build && yarn start` so that if `build` fails, the command fails?